config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  post '/guests/:id/rooms', to: 'guests#create'
```

Comment:
Overal this route is fine, but see my above comment on which controller this should be routed to.